beeflow/common/api.py

```diff
@@ -0,0 +1,12 @@
+"""BeeApi wrapper around Flask-Restful for handling exceptions."""
+import traceback
+from flask import make_response, jsonify
+from flask_restful import Api
+
+
+class BeeApi(Api):
+    """Wrapper around Flask-Restful's API to catch exceptions."""
+
+    def handle_error(self, e):  # noqa (conflict on naming in base class vs. following convention)
```

Comment:
When does this function in the wrapper get called? It's not immediately obvious to me how an exception would trigger it.

Reply:
Huh cool